module/config.mjs

```diff
@@ -5316,11 +5316,25 @@ function addPower(powerDescription6e, powerOverrideFor5e) {
         {
             key: "TRANSFORM",
             type: ["attack", "standard"],
-            behaviors: ["attack"],
+            behaviors: ["attack", "dice"],
             perceivability: "obvious",
             duration: "instant",
             target: "target's DCV",
             range: HERO.RANGE_TYPES.STANDARD,
+            costPerLevel: function(item) {
```

Comment:
Not sure this is correct for 5e as it only has 3 levels I believe. Please confirm and use the `is5e` optional boolean to determine which system the item is associated with.

Reply:
ah thanks for that; I'll snag the 5e rulebook to double check everything

Reply:
This should be fixed now